### PR TITLE
Update install docs

### PR DESCRIPTION
**Pull request by evanxg852000**
### Description
Closes https://github.com/quickwit-inc/quickwit/issues/875

### How was this PR tested?
run through the doc by executing commands.

### CODE REVIEW

File: docs/getting-started/installation.md [hunk 1]
```diff
@@ -55,27 +66,31 @@ To get the full gist of this, let's run a minified version of the - [Quickstart
 mkdir data && cd data
 
 # download wikipedia dataset files
-curl -o wikipedia_index_config.json https://raw.githubusercontent.com/quickwit-inc/quickwit/main/examples/index_configs/wikipedia_index_config.json
+curl -o wikipedia_index_config.yaml https://raw.githubusercontent.com/quickwit-inc/quickwit/main/config/tutorials/wikipedia/index-config.yaml
 curl -o wiki-articles-10000.json https://quickwit-datasets-public.s3.amazonaws.com/wiki-articles-10000.json
+```
+
+To expose the rest API outside of the docker container, we need to customize the default configuration provided by Quickwit. Download it, uncomment and update the searcher config section by setting the `rest_listen_address` to `0.0.0.0`.
```
Comment (fmassot) [inline docs/getting-started/installation.md:73]: Would be nice to be able to provide an config file with the right `rest_listen_address`. 
What I would do is to put a `quickwit_docker.yaml` with `rest_listen_address` to `0.0.0.0`.
Then we can just tell the user to download the file.

File: docs/getting-started/installation.md [hunk 2]
```diff
@@ -55,27 +66,31 @@ To get the full gist of this, let's run a minified version of the - [Quickstart
...
+curl -o quickwit.yaml https://raw.githubusercontent.com/quickwit-inc/quickwit/main/config/quickwit.yaml
+```
+
+```bash
 # create, index and search using the container 
-docker run -v "$(pwd)":"/usr/quickwit" quickwit/quickwit-cli new --index-uri file:///usr/quickwit/wikipedia --index-config-path wikipedia_index_config.json
-docker run -v "$(pwd)":"/usr/quickwit" quickwit/quickwit-cli index --index-uri file:///usr/quickwit/wikipedia --input-path wiki-articles-10000.json
-docker run -v "$(pwd)":"/usr/quickwit" quickwit/quickwit-cli search --index-uri file:///usr/quickwit/wikipedia --query "barack obama"
-docker run -v "$(pwd)":"/usr/quickwit" -p 8080:8080 quickwit/quickwit-cli serve --index-uri file:///usr/quickwit/wikipedia 
+docker run -v "$(pwd)":"/quickwit/qwdata" quickwit/quickwit index create --index=wiki --index-config=./qwdata/wikipedia_index_config.yaml
```
Comment (fmassot) [inline docs/getting-started/installation.md:81]: let's be consistent and keep wikipedia as index id

File: docs/getting-started/quickstart.md [hunk 3]
```diff
@@ -39,108 +40,100 @@ Let's create an index configured to receive Wikipedia articles.
...
+      tokenizer: default
+      record: position
+      stored: true
+    - name: body
+      type: text
+      tokenizer: default
+      record: position
+      stored: true
+    - name: url
+      type: textd.
```
Comment (fmassot) [inline docs/getting-started/quickstart.md:66]: typo `type: textd.`

File: docs/getting-started/quickstart.md [hunk 4]
```diff
@@ -39,108 +40,100 @@ Let's create an index configured to receive Wikipedia articles.
...
+      stored: true # Field is stored. 
+
+search_settings:
+  # If you do not specify fields in your query, those fields will be used.
+  default_search_fields: [title, body]
 ```
 
-Now we can create the index with the command:
+Let's make use of the default configuration file provided in quickwit installation throughout this guide.
+by setting the `QW_CONFIG` environement variable.
```
Comment (fmassot) [inline docs/getting-started/quickstart.md:76]: typo `environement`

File: docs/getting-started/quickstart.md [hunk 5]
```diff
@@ -39,108 +40,100 @@ Let's create an index configured to receive Wikipedia articles.
...
 # Index our 10k documents.
-./quickwit index --index wikipedia --data-dir-path ./wikipedia --metastore-uri file://$(pwd)/metastore --input-path wiki-articles-10000.json
+./quickwit index ingest --index=wikipedia --input-path=./wiki-articles-10000.json
 ```
 
 Wait one second or two and check if it worked by using `search` command:
 
 ```bash
-./quickwit search --index wikipedia --metastore-uri file://$(pwd)/metastore --query "barack AND obama"
+./quickwit index search --index=wikipedia --query "barack AND obama"
```
Comment (fmassot) [inline docs/getting-started/quickstart.md:108]:
```suggestion
./quickwit index search --index wikipedia --query "barack AND obama"
```

File: docs/getting-started/quickstart.md [hunk 6]
```diff
@@ -151,21 +144,24 @@ Don't forget to encode correctly the query params to avoid bad request (status 4
 Let's do some cleanup by deleting the index:
 
 ```bash
-./quickwit delete --index wikipedia --metastore-uri file:///$(pwd)/metastore
+./quickwit index delete --index=wikipedia
```
Comment (fmassot) [inline docs/getting-started/quickstart.md:147]:
```suggestion
./quickwit index delete --index wikipedia
```

File: docs/getting-started/quickstart.md [hunk 7]
```diff
@@ -151,21 +144,24 @@ Don't forget to encode correctly the query params to avoid bad request (status 4
...
 ## TLDR
 
+Run the following command from within Quickwit's installation directory.
+
 ```bash
-curl -o wikipedia_index_config.json https://raw.githubusercontent.com/quickwit-inc/quickwit/main/examples/index_configs/wikipedia_index_config.json
-./quickwit new --index-uri file://$(pwd)/wikipedia --index-config-path ./wikipedia_index_config.json --index wikipedia --metastore-uri file://$(pwd)/metastore/
+curl -o wikipedia_index_config.yaml https://raw.githubusercontent.com/quickwit-inc/quickwit/main/config/tutorials/wikipedia/index-config.yaml
+export QW_CONFIG=./config/quickwit.yaml
+./quickwit index create --index=wikipedia --index-config=./wikipedia_index_config.yaml
```
Comment (fmassot) [inline docs/getting-started/quickstart.md:160]:
```suggestion
./quickwit index create --index wikipedia --index-config /wikipedia_index_config.yaml
```

File: docs/getting-started/installation.md [hunk 8]
```diff
@@ -9,42 +9,53 @@ We notably provide musl builds to provide static binaries with no dependencies.
...
 ```bash
 curl -L https://install.quickwit.io | sh
-./quickwit --version
 ```
 
-## Build with cargo
+All this script does is download the correct binary archive for your machine and extract it in the current working directory. This means you can download any desired archive from [github](https://github.com/quickwit-inc/quickwit/releases) that match your OS architecture and manually extract it anywhere.
 
-If cargo is on your system, you can install the package from crates.io.
+Once installed or extracted, All Quickwit's installation files can be found in a directory named `quickwit-{version}` where version is the corresponding version of Quickwit. This directory has the following layout:
```
Comment (fmassot) [inline docs/getting-started/installation.md:33]: `All Quickwit's installation` -> `all Quickwit's installation`

Comment (fmassot) [reply docs/getting-started/installation.md:33]: where version is -> where `version` is

File: docs/getting-started/installation.md [hunk 9]
```diff
@@ -9,42 +9,53 @@ We notably provide musl builds to provide static binaries with no dependencies.
...
 
 If you use docker, this might be one of the quickest way to get going. 
-The following command will pull the image from [dockerhub](https://hub.docker.com/r/quickwit/quickwit-cli)
+The following command will pull the image from [dockerhub](https://hub.docker.com/r/quickwit/quickwit)
 and gets you right in the shell of the running container ready to execute Quickwit commands. 
 Note that we are also mounting the working directory as volume. This is useful when you already have your dataset ready on your machine and want to work with Quickwit docker image.
 
 ```bash
-docker run -it -v "$(pwd)":"/usr/quickwit" --entrypoint ash quickwit/quickwit-cli
+docker run -it -v "$(pwd)":"/quickwit/my-data" --entrypoint ash quickwit/quickwit
```
Comment (fmassot) [inline docs/getting-started/installation.md:58]: `/my-data` naming is confusing

**Review by fmassot — CHANGES_REQUESTED** [review]
See comemnts

File: docs/getting-started/installation.md (hunk 1 above)
Comment (evanxg852000) [reply docs/getting-started/installation.md:73]: I thought about it but, I was like it would be many files. I should have pinged you about it.
Thanks I will do that.

File: docs/getting-started/quickstart.md [hunk 10]
```diff
@@ -39,108 +40,100 @@ Let's create an index configured to receive Wikipedia articles.
...
+      tokenizer: default
+      record: position
+      stored: true
+    - name: body
+      type: text
+      tokenizer: default
+      record: position
+      stored: true
+    - name: url
+      type: text
```
Comment (fmassot) [inline docs/getting-started/quickstart.md:66]: stored is true by default, don't need to add it here.

File: docs/getting-started/quickstart.md [hunk 11]
```diff
@@ -39,108 +40,100 @@ Let's create an index configured to receive Wikipedia articles.
...
+      tokenizer: raw
+      stored: true # Field is stored. 
+
+search_settings:
+  # If you do not specify fields in your query, those fields will be used.
+  default_search_fields: [title, body]
 ```
 
-Now we can create the index with the command:
+Let's make use of the default configuration file provided in quickwit installation throughout this guide.
```
Comment (fmassot) [inline docs/getting-started/quickstart.md:75]: "by setting the" starts a new sentence, it seems that it's the continuation of the previous.

File: docs/getting-started/quickstart.md [hunk 12]
```diff
@@ -39,108 +40,100 @@ Let's create an index configured to receive Wikipedia articles.
...
-To simplify our command, we now assume that the index and metastore are to be created in the current directory and use the `pwd` command to specify the index uri:
+Now we can create the index with the command:
 
 ```bash
-./quickwit new --index-uri file://$(pwd)/wikipedia --index-config-path ./wikipedia_index_config.json --index wikipedia --metastore-uri file://$(pwd)/metastore/
+./quickwit index create --index wikipedia --index-config=./wikipedia_index_config.yaml
 ```
 
-Check that an empty directory `$(pwd)/wikipedia` has been created, Quickwit will write index files here and a `quickwit.json` which contains the [index metadata](../overview/architecture.md#index-metadata).
+Check that a directory `$(pwd)/qwdata/wikipedia` has been created, Quickwit will write index files here and a `quickwit.json` which contains the [index metadata](../overview/architecture.md#index-metadata).
```
Comment (fmassot) [inline docs/getting-started/quickstart.md:88]: I prefer the less verbone path `./qwdata/wikipedia`

File: docs/getting-started/quickstart.md [hunk 13]
```diff
@@ -39,108 +40,100 @@ Let's create an index configured to receive Wikipedia articles.
...
+Quickwit can index data from many [sources](./sources.md). We will use a new line delimited json [ndjson](http://ndjson.org/) datasets as our data source.
 Let's download [a bunch of wikipedia articles (10 000)](https://quickwit-datasets-public.s3.amazonaws.com/wiki-articles-10000.json) in [ndjson](http://ndjson.org/) format and index it.
 
 ```bash
 # Download the first 10_000 Wikipedia articles.
 curl -o wiki-articles-10000.json https://quickwit-datasets-public.s3.amazonaws.com/wiki-articles-10000.json
 
 # Index our 10k documents.
-./quickwit index --index wikipedia --data-dir-path ./wikipedia --metastore-uri file://$(pwd)/metastore --input-path wiki-articles-10000.json
+./quickwit index ingest --index wikipedia --input-path=./wiki-articles-10000.json
```
Comment (fmassot) [inline docs/getting-started/quickstart.md:102]: can you make the command consistent everywhere? there are some `arg=value` with `=` here and there

**Review by fmassot — APPROVED** [review]
Small comments that you can easily fix before merging, approved!

File: docs/getting-started/quickstart.md (hunk 10 above)
Comment (evanxg852000) [reply docs/getting-started/quickstart.md:66]: I wanted it to be exactly what we have from the download `config/tutorials/wikipedia/index-config.yaml` I will update that one too.
